app/services/rss_reader.rb

```diff
@@ -121,7 +121,8 @@ def article_exists?(user, item)
     relation.where(title: title).or(relation.where(feed_source_url: feed_source_url)).exists?
   end
 
-  def log_error(error_msg, metadata)
-    Rails.logger.error(error_msg, metadata)
+  # TODO: [@thepracticaldev/oss]: How did the metadata argument to Rails.logger work before?
```

Comment:
See the PR description. I didn't want to spend too much time on this, so just removed metadata logging for now (which is also the reason for the extra debug output above).